repository/p.json

```diff
@@ -1387,6 +1387,17 @@
 				}
 			]
 		},
+		{
+			"name": "Pipenv — Python Dev Workflow for Humans",
+			"details": "https://github.com/kennethreitz/pipenv-sublime",
+			"labels": ["python", "pip", "virtualenv", "pipenv", "Pipfile"],
```

Comment:
`pipenv` and `Pipfile` should probably be removed since it is unlikely there will be more packages with those tags

Reply:
Imo those can stay, as some other package might provide something to work with pipenv or with Pipfiles.

Reply:
(And there we go, a package like this has already been submitted: https://github.com/wbond/package_control_channel/pull/7007/files#diff-a19631c4b751bdd943200931c7e6b81cR2480)